docker/ci/test_scripts.sh

```diff
@@ -9,11 +9,14 @@ container="$1"
 dir=$(dirname "${BASH_SOURCE[0]}")
 source "${dir}/check_service_status.sh"
 
+# Output the version file for debugging
+docker compose exec "$container" cat /var/www/circulation/src/palace/manager/_version.py
+
 # Wait for container to start
 wait_for_runit "$container"
 
 # Make sure database initialization completed successfully
-timeout 240s grep -q 'Initialization complete' <(docker compose logs "$container" -f 2>&1)
+timeout 240s grep -q -e 'Initialization complete' -e "Migrations complete" <(docker compose logs "$container" -f 2>&1)
```

Comment:
I needed to change this, because there is a race condition now that we test both webapp and scripts in the same container, previously we would have always hit initialization, now the first container to start does it.